step/serialization.md

```diff
@@ -137,68 +146,69 @@ public class ClassSerializationClient {
 }
 </copy>
 ```
-To execute the application, first compile the `MessageClass`:
+
+To execute the application, first compile the `Message`:
 
 ```
 <copy>
-javac MessageClass.java
+javac Message.java
 </copy>
 ```
 
 Next start the "Server":
 
 ```
 <copy>
-java ClassSerializationServer.java
+java SerializationServer.java
```

Comment:
you should either launch the server in background or ask to use 2 different shells (preferred)